server/src/test/integration/org/apache/cassandra/sidecar/testing/bytebuddy/BBHelperMovingNode.java

```diff
@@ -49,15 +50,48 @@ public static void install(ClassLoader cl, int nodeNumber, int movingNodeIndex)
         if (nodeNumber == movingNodeIndex)
         {
             TypePool typePool = TypePool.Default.of(cl);
-            TypeDescription description = typePool.describe("org.apache.cassandra.service.RangeRelocator")
-                                                  .resolve();
-            new ByteBuddy().rebase(description, ClassFileLocator.ForClassLoader.of(cl))
-                           .method(named("stream"))
+
+            if (installTcm(cl, typePool)) {
```

Comment:
curly bracket in the next line? I noticed a few in other places too. Can you please reformat them according to the coding conventions of this repo?